ctru-rs/src/services/gspgpu.rs

```diff
@@ -1,31 +1,31 @@
 //! GSPGPU service
 
-use std::convert::From;
-
 #[derive(Copy, Clone, Debug)]
+#[repr(u32)]
 pub enum Event {
```

Comment:
I wonder if we should also add `#[non_exhaustive]` on any of these enums? It might be hard to say at a glance which ones might have stuff added or not

Reply:
Unless there are specific changes in `libctru`(which we cannot predict) there shouldn't be any major changes in the enums already implemented. `#[non_exhaustive]` would be a substantial limit to the possibilities enabled by the public enums.